### PR TITLE
black: Apply to tests

### PR DESCRIPTION
**Pull request by evanpurkhiser**
I don't expect to merge this right away as is, since there's a lot of formatting I think black might do that's not optimal.

I'll try and add some comment call out for weird things.

**However**, I do think it will be pretty safe to merge this without so many small PRs since

1. Test code is not run in production, so the chance that something is broken that is shipped to production is pretty much zero.
2. If it runs in CI, the code works. There won't be any rogue syntax errors if CI doesn't fail.

### CODE REVIEW

**Comment by evanpurkhiser** [comment]
Have to run to an invterview. Will try and get the last bit of PRs up after. Thank you for reviewing everything @kiorky !!

I will start looking at the DST bugs today as well.

File: src/croniter/tests/test_croniter.py [hunk 1]
```diff
@@ -316,17 +343,19 @@ def testPrevLastDayOfMonth(self):
...
-        self.assertRaises(ValueError, croniter, '* * 5-100 * *')
+        self.assertRaises(ValueError, croniter, "* * * *")
+        self.assertRaises(ValueError, croniter, "-90 * * * *")
+        self.assertRaises(ValueError, croniter, "a * * * *")
+        self.assertRaises(ValueError, croniter, "* * * janu-jun *")
+        self.assertRaises(ValueError, croniter, "1-1_0 * * * *")
+        self.assertRaises(ValueError, croniter, "0-10/error * * * *")
+        self.assertRaises(ValueError, croniter, "0-10/ * * * *")
+        self.assertRaises(
+            CroniterBadCronError, croniter, "0-1& * * * *", datetime.now()
```
Comment (kiorky) [inline src/croniter/tests/test_croniter.py:356]: linelength 120 should fix spurious linereturn.

File: src/croniter/tests/test_croniter.py [hunk 2]
```diff
@@ -340,39 +369,45 @@ def testSundayToThursdayWithAlphaConversion(self):
...
+        self.assertEqual(croniter("0-59 0 1 1 0").expanded[m], wildcard)
+        self.assertEqual(croniter("0 0-23 1 1 0").expanded[h], wildcard)
+        self.assertEqual(croniter("0 0 1-31 1 0").expanded[dow], [0])
+        self.assertEqual(croniter("0 0 1-31 1 *").expanded[d], wildcard)
+        self.assertEqual(croniter("0 0 1 1-12 0").expanded[mon], wildcard)
+        self.assertEqual(croniter("0 0 1 1 0-6").expanded[dow], [0, 1, 2, 3, 4, 5, 6])
+        self.assertEqual(croniter("0 0 * 1 0-6").expanded[dow], wildcard)
+        self.assertEqual(croniter("0 0 1 1 0-6").expanded[dow], [0, 1, 2, 3, 4, 5, 6])
+        self.assertEqual(
+            croniter("0 0 1 1 0-6,sat#3").expanded[dow], [0, 1, 2, 3, 4, 5, 6]
```
Comment (kiorky) [inline src/croniter/tests/test_croniter.py:385]: idem

File: src/croniter/tests/test_croniter.py [hunk 3]
```diff
@@ -1967,124 +2205,783 @@ def test_get_next_update_current(self):
...
-                                 [2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16,
-                                  17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31])
+        self._test_sunday_ranges(
+            "0 0 * * Sun-Sun",
+            [
+                2,
+                3,
+                4,
+                5,
+                6,
```
Comment (kiorky) [inline src/croniter/tests/test_croniter.py:2265]: those one should not be reformated as it is unreadable as is :)

File: src/croniter/tests/test_croniter.py [hunk 4]
```diff
@@ -1967,124 +2205,783 @@ def test_get_next_update_current(self):
...
+            ],
+        )
+        self._test_mth_cron_ranges(
+            "0 0 1 Apr-Mar/2 *",
+            [
+                "24 3",
+                "24 4",
+                "24 6",
+                "24 8",
+                "24 10",
```
Comment (kiorky) [inline src/croniter/tests/test_croniter.py:2891]: idem

File: src/croniter/tests/test_croniter_random.py [hunk 5]
```diff
@@ -9,7 +9,7 @@ class CroniterRandomTest(base.TestCase):
 
     def test_random(self):
         """Test random definition"""
```
Comment (kiorky) [inline src/croniter/tests/test_croniter_random.py:11]: whole file ok.

File: src/croniter/tests/test_croniter_range.py [hunk 6]
```diff
@@ -4,7 +4,13 @@
 import unittest
 from datetime import datetime, timedelta
 import pytz
-from croniter import croniter, croniter_range, CroniterBadDateError, CroniterBadCronError, CroniterBadTypeRangeError
+from croniter import (
```
Comment (kiorky) [inline src/croniter/tests/test_croniter_range.py:7]: whole file ok.

File: src/croniter/tests/test_speed.py [hunk 7]
```diff
@@ -10,9 +10,9 @@
 
 class timerTest(object):
     def __init__(self):
-        self.tests = tuple(getattr(self, m)
```
Comment (kiorky) [inline src/croniter/tests/test_speed.py:13]: whole file ok.

File: src/croniter/tests/base.py [hunk 8]
```diff
@@ -5,9 +5,10 @@
 
 
 class TestCase(unittest.TestCase):
-    '''
```
Comment (kiorky) [inline src/croniter/tests/base.py:8]: whole file ok.

**Comment by kiorky** [comment]
> Have to run to an invterview. Will try and get the last bit of PRs up after. Thank you for reviewing everything @kiorky !!
> 
> I will start looking at the DST bugs today as well.

ok, just to let you know that i'm on `CET` timezone so i ll be off soon, and will look then tomorrow. lots of thanks too for your work here !

**Comment by evanpurkhiser** [comment]
I'll at least split this into things like quotes 👍 and turn some formatting off for the egregious lists

**Comment by kiorky** [comment]
Do you see anything left to do for tests formatting ?

**Comment by evanpurkhiser** [comment]
Nah this sohuld be done.

**Pull request closed without merging**